extension/public/worker/twitter.js

```diff
@@ -0,0 +1,14 @@
+/** @param {HTMLDivElement} el */
+function checkTweet(el) {
+  if (window.matchesBlocklist(el.innerText)) {
+    // eslint-disable-next-line no-param-reassign
+    el.style.filter = "blur(2px)";
```

Comment:
Are you able to make the blur slightly stronger? I can kind of make out what the tweet says here. 

![image](https://user-images.githubusercontent.com/78716153/192125307-c93286b9-5ed4-4d8b-86c3-6d2ce1a89300.png)